hkpy/hkbase/observer/clients/observerclient.py

```diff
@@ -0,0 +1,61 @@
+###
+# Copyright (c) 2019-present, IBM Research
+# Licensed under The MIT License [see LICENSE for details]
+###
+
+from typing import TypeVar
+HKBase = TypeVar('HKBase')
+
+
+class ObserverClient:
```

Comment:
Abstract class?

Reply:
not necessarily

Reply:
Why though? By itself, an object of this type cannot observe anything.

Reply:
Fair point, I'll standardize ObserverClient and ConfigurableObserverClient as abstract classes in both libs